From: Net-SNMP Packaging Team <pkg-net-snmp-devel@lists.alioth.debian.org>
Date: Thu, 18 Jun 2015 06:12:05 +0900
Subject: fix_regular_expression

>>> import os
>>> import re
>>> import string
>>> import sys
>>> netsnmp_libs="-Wl,-z,relro -Wl,-z,now -L/usr/lib/x86_64-linux-gnu -lnetsnmp -lcrypto -lm"
>>> print netsnmp_libs
-Wl,-z,relro -Wl,-z,now -L/usr/lib/x86_64-linux-gnu -lnetsnmp -lcrypto -lm
>>> re.findall(r"-l(\S+)", netsnmp_libs)
['inux-gnu', 'netsnmp', 'crypto', 'm']

 Just inserting space will fix this.
>>> re.findall(r" -l(\S+)", netsnmp_libs)
['netsnmp', 'crypto', 'm']

Author: Hideki Yamane <henrich@debian.org>
---
 python/setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index c520df9..0bd3155 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -18,14 +18,14 @@ if intree:
     netsnmp_libs = os.popen(basedir+'/net-snmp-config --libs').read()
     libdir = os.popen(basedir+'/net-snmp-config --build-lib-dirs '+basedir).read()
     incdir = os.popen(basedir+'/net-snmp-config --build-includes '+basedir).read()
-    libs = re.findall(r"-l(\S+)", netsnmp_libs)
+    libs = re.findall(r" -l(\S+)", netsnmp_libs)
     libdirs = re.findall(r"-L(\S+)", libdir)
     incdirs = re.findall(r"-I(\S+)", incdir)
 else:
     netsnmp_libs = os.popen('net-snmp-config --libs').read()
     libdirs = re.findall(r"-L(\S+)", netsnmp_libs)
     incdirs = []
-    libs = re.findall(r"-l(\S+)", netsnmp_libs)
+    libs = re.findall(r" -l(\S+)", netsnmp_libs)
 
 setup(
     name="netsnmp-python", version="1.0a1",
